bin/ilamb-run

```diff
@@ -187,6 +187,48 @@ def ParseModelSetup(model_setup,models=[],verbose=False,filter="",regex="",model
 
     return M
 
+def InitializeDrainageBasins(filenames):
```

Comment:
Rename this function, you are using shapefiles for drainage bins, but generically people could use them for anything and the name should reflect that.